.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "Frameworks/Prelude"]
 	path = Frameworks/Prelude
 	url = https://github.com/kickstarter/Kickstarter-Prelude.git
+[submodule "Frameworks/native-secrets"]
+	path = Frameworks/native-secrets
+	url = https://github.com/kickstarter/native-secrets.git
```

Comment:
this too. we decided to drop the `native-secrets` submodule and do a simple clone in the bootstrap.